eq-author/src/components/CommentAccordion/index.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+// import { kebabCase } from "lodash";
+import styled from "styled-components";
+import { colors } from "constants/theme";
+import chevron from "./icon-chevron.svg";
+
+const Header = styled.div`
+  padding-left: 1em;
+  color: ${colors.blue};
+  text-decoration: underline;
+  &:hover {
+    /* background: #f3f3f3; */
+    color: ${colors.grey};
+  }
+`;
+
+export const Title = styled.h2`
+  font-size: 0.75em;
+  letter-spacing: 0.05em;
+  vertical-align: middle;
+
+  text-align: left;
+
+  margin: 0;
+  padding: 0.5em 0;
+  position: relative;
+`;
+
+export const Body = styled.div`
+  overflow: hidden;
+  transition: opacity 100ms ease-in-out;
+  opacity: ${props => (props.open ? "1" : "0")};
+  height: ${props => (props.open ? "auto" : "0")};
+`;
+
+const Container = styled.div`
+  /* fixes ghosting issue in Chrome */
+  backface-visibility: hidden;
+  &:last-of-type ${Body} {
+    border-bottom: 1px solid ${colors.lightMediumGrey};
+  }
+`;
```

Comment:
Feel free to ignore this as it's not part of this PR, but as Henning fixed the ghosting issue I reckon we could remove this; not sure what bit is intended to fix the ghosting issue, though, so might need to have a play.

Reply:
well spotted, mixes the rougue underline also!